lib/passageidentity/user_api.rb

```diff
@@ -191,7 +191,24 @@ def delete(user_id:)
       end
     end
 
+    def revoke_device(user_id:, device_id:)
```

Comment:
either this can just call `delete_device` directly or vice versa to avoid the duplication